src/foam/u2/crunch/wizardflow/SkipGrantedAgent.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.crunch.wizardflow',
+  name: 'SkipGrantedAgent',
+  documentation: `
+    Allows filtering or skipping of granted capabilities. Also exports a wizard
+    position in case wizardlets are to be skipped.
+  `,
+  imports: [ 'wizardlets' ],
+  exports: [ 'initialPosition' ],
+  requires: [
+    'foam.nanos.crunch.CapabilityJunctionStatus',
+    'foam.nanos.crunch.ui.CapabilityWizardlet',
+    'foam.u2.wizard.WizardPosition'
+  ],
+
+  enums: [
+    {
+      name: 'SkipMode',
+      values: ['HIDE','SHOW','SKIP']
```

Comment:
Change values to objects and add documentation properties

Reply:
Done.